change_selection_down.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+todo_task_file=$1
+lines=$( wc -l < "$todo_task_file" )
+i=$lines
+
+while ((i > 0)); do
+  line_number=$i
+  task_line="$( sed "${line_number}q;d" "$todo_task_file" )"
+  task_status="$(echo $task_line| cut -d']' -f 1)"
+  task_text="$(echo $task_line| cut -d']' -f 2)"
+
+  if [[ "$task_status" == "[>" ]] && ((line_number < lines)); then
+    # Unselect the selected line
+    unselect_task_update="[<]${task_text}"
+    sed -i '' "${line_number} s/.*/${unselect_task_update}/" "$todo_task_file"
+    
+    # Select the next line
+    next_line_number=$((line_number+1))
+    next_task_line="$( sed "${next_line_number}q;d" "$todo_task_file" )"
+    next_task_text="$(echo $next_task_line| cut -d']' -f 2)"
+    select_task_update="[>]${next_task_text}"
```

Comment:
I think it would be nice if this wrapped at the top/bottom

Reply:
I'm not sure I understand what you mean here 😅

Reply:
If you're at the bottom line and press down you should go to the first line and vice versa. :)

Reply:
Aha! Sure :)